utils/download_test.go

```diff
@@ -0,0 +1,231 @@
+package utils
+
+import (
+	"errors"
+	"io/ioutil"
+
```

Comment:
This new line isnt needed. Removing it will keep all internal library imports together